### DISCUSSION TITLE
建议弄一个题库整合

### DISCUSSION THREAD

**Discussion by qixishunzi** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#25] [Ideas]
目前有同一个问题，小米那边是随机抽五个选项给你来做。同一个问题，可以把关于这个题所有选项整合一下

**Comment by MlgmXyysd** [comment]
小米天天乱改反转，整合起来不是很方便的样子，考虑加到知识库中

**Comment by MlgmXyysd** [comment]
https://xmbl.neko.ink/?action=problem_list
